scripts/consume_sitemaps.py

```diff
@@ -11,18 +14,22 @@
 import isb_web
 import isb_web.config
 import logging
+import re
+
+from isb_lib.models.thing import Thing
 from isb_lib.sitemaps.sitemap_fetcher import (
     SitemapIndexFetcher,
     SitemapFileFetcher,
-    ThingFetcher,
+    ThingFetcher, ThingsFetcher,
 )
 from isb_web import sqlmodel_database
-from isb_web.sqlmodel_database import SQLModelDAO
+from isb_web.sqlmodel_database import SQLModelDAO, all_thing_identifiers, thing_identifiers_from_resolved_content
 
-CONCURRENT_DOWNLOADS = 50
+CONCURRENT_DOWNLOADS = 50000
 # when we hit this length, add some more to the queue
 REFETCH_LENGTH = CONCURRENT_DOWNLOADS / 2
 
+__NUM_THINGS_FETCHED = 0
```

Comment:
@datadavev -- I ended up using a module global for this as it was convenient and easy to do so.  Is that a red flag?  I saw varying opinions online about whether this was acceptable Python hygiene or not :)